user/views.py

```diff
@@ -4,8 +4,57 @@
 from django.http      import JsonResponse
 
 from .models          import User
+from .models          import Product
+from .models          import WishList
+from user.utils       import login_required
 from weline.settings  import SECRET_KEY, ALGORITHM_KEY
 
+class WishListView(View):
+    @login_required
+    def get(self, request):
+        wish_list = []
+        user_wishlist = User.objects.get(id = request.user.id).wish_list.all()
```

Comment:
```suggestion
        user_wishlist =request.user.wish_list.all()
```
이미 유저 객체가 있기 때문에 그 객체를 다시 get 해오시지는 않으셔도 괜찮습니다